### DISCUSSION TITLE
Are people allowed to use things like the hud as long as they credit you

### DISCUSSION THREAD

**Discussion by helloIlovegimkit** [TheLazySquid/GimkitCheat#62] [General]
this idea came to me since I'm trying to make a quizizz cheat @TheLazySquid  or if you could just provide the method or if u used a vid provide that so I can learn how to make a hud like that since it is clean

**Comment by TheLazySquid** [comment]
Feel free to use the hud, as long as you give some credit. It uses Svelte, so I recommend using a bundler such as [rollup](https://rollupjs.org/) with a plugin such as [rollup-plugin-svelte](https://www.npmjs.com/package/rollup-plugin-svelte).

**Reply by helloIlovegimkit** [reply]
thank you
